docs/reference/sql/functions/aggs.asciidoc

```diff
@@ -397,6 +397,16 @@ https://en.wikipedia.org/wiki/Kurtosis[Quantify] the shape of the distribution o
 include-tagged::{sql-specs}/docs/docs.csv-spec[aggKurtosis]
```

Comment:
I think it's worth adding examples in the docs (either a dedicated one or for most aggregations) that shows aggregations alongside a scalar function as it might not be obvious, especially for existing users.

This would also address @astefan point of having a test that can be easily be verified.